python/ray/worker.py

```diff
@@ -674,6 +688,50 @@ def export_remote_function(self, function_id, function_name, function,
             })
         self.redis_client.rpush("Exports", key)
 
+    def register_class_on_all_workers(self, function):
```

Comment:
What is the difference between this code path and the "run function on all workers" code path? Why separate them?